rpcs3/Emu/RSX/VK/VKPipelineCompiler.h

```diff
@@ -119,9 +119,9 @@ namespace vk
 				VkShaderModule modules[2],
 				const std::vector<glsl::program_input>& vs_in,
 				const std::vector<glsl::program_input>& fs_in,
-				callback_t func)
+			    callback_t func)
+			    : callback_func(func)

```

Comment:
You broke some indentation)

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/f8a86a38906e0cdfd0cfdcc3d86c32d1eabe084b